### PR TITLE
website: Add consent manager

### PR DESCRIPTION
**Pull request by EnMod**
[:mag: Preview link](https://terraform-website-git-nqweb-add-demandbase-consent-hashicorp.vercel.app/)

---

This PR re-enables the consent manager for the Terraform website. This also adds a Demandbase tag to the consent manager.

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -19,6 +20,7 @@ import alertBannerData, { ALERT_BANNER_ACTIVE } from 'data/alert-banner'
 NProgress({ Router })
 // const { ConsentManager, openConsentManager } = createConsentManager({
 //   preset: 'oss',
+//   otherServices: [...localConsentManagerServices]
 // })
```
Comment (EnMod) [inline pages/_app.js:24]: Hey @dstaley , was there a reason the consent manager for Terraform was disabled? I'll leave it be if so!

Comment (dstaley) [reply pages/_app.js:24]: It was originally disabled to give the TF team time to prepare for their analytics to get all wonky. Feel free to turn it on now that we're past that point!

File: pages/_app.js [hunk 2]
```diff
@@ -47,8 +49,8 @@ function App({ Component, pageProps }) {
       <div className="page-content">
         <Component {...pageProps} />
       </div>
-      {/* <Footer openConsentManager={openConsentManager} />
-      <ConsentManager /> */}
+      <Footer openConsentManager={openConsentManager} />
+      <ConsentManager />
       <Footer />
```
Comment (alexcarpenter) [inline pages/_app.js:54]:
```suggestion

```

**Review by alexcarpenter — CHANGES_REQUESTED** [review]
Duplicate footers

**Review by alexcarpenter — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**